### PR TITLE
Procedural Prefabs: Add example LOD script

### PR DESCRIPTION
**Pull request by amzn-mike**

### CODE REVIEW

File: AutomatedTesting/Editor/Scripts/scene_helpers.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import traceback, logging, json
```
Comment (AMZN-stankowi) [inline AutomatedTesting/Editor/Scripts/scene_helpers.py:1]: this needs the legal header

File: AutomatedTesting/Editor/Scripts/scene_helpers.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    if output is not None and output.IsSuccess():
+        json_string = output.GetValue()
+        uuid = azlmbr.math.Uuid_CreateRandom().ToString()
+        json_result = json.loads(json_string)
+        # Add a PrefabGroup to the manifest and store the JSON on it
+        scene_manifest.add_prefab_group(prefab_name, uuid, json_result)
+    else:
+        raise RuntimeError(
+            "SaveTemplateToString failed for template id {}, prefab {}".format(created_template_id, prefab_filename))
```
Comment (AMZN-stankowi) [inline AutomatedTesting/Editor/Scripts/scene_helpers.py:68]: you might want to add a newline to the end of the file. I think that's more Linux friendly or something.

Comment (amzn-mike) [reply AutomatedTesting/Editor/Scripts/scene_helpers.py:68]: I think there is one.  Pycharm is showing it anyway, and it normally complains about not having one.

File: AutomatedTesting/Editor/Scripts/scene_helpers.py (hunk 1 above)
Comment (amzn-mike) [reply AutomatedTesting/Editor/Scripts/scene_helpers.py:1]: Added

File: AutomatedTesting/Editor/Scripts/auto_lod.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                scene_manifest.lod_unselect_node(lod, y)
+
+    # Create an editor entity
+    entity_id = azlmbr.entity.EntityUtilityBus(azlmbr.bus.Broadcast, "CreateEditorReadyEntity", mesh_group_name)
+    # Add an EditorMeshComponent to the entity
+    editor_mesh_component = azlmbr.entity.EntityUtilityBus(azlmbr.bus.Broadcast, "GetOrAddComponentByTypeName", entity_id, "AZ::Render::EditorMeshComponent")
+    # Set the ModelAsset assetHint to the relative path of the input asset + the name of the MeshGroup we just created + the azmodel extension
+    # The MeshGroup we created will be output as a product in the asset's path named mesh_group_name.azmodel
+    # The assetHint will be converted to an AssetId later during prefab loading
+    json_update = json.dumps({
```
Comment (jackalbe) [inline AutomatedTesting/Editor/Scripts/auto_lod.py:84]: does this update the component LOD or is this simply the same code to assign a mesh component to an entity?

Comment (amzn-mike) [reply AutomatedTesting/Editor/Scripts/auto_lod.py:84]: Its the same.  The scene manifest has the LOD settings, the Mesh component does have some more LOD configuration on it but I just left it default.

**Review by jackalbe — APPROVED** [review]

**Pull request merged**